lib/api/client-commands/logs/getLog.js

```diff
@@ -0,0 +1,54 @@
+const ClientCommand = require('../_base-command.js');
```

Comment:
I'm thinking that it would be better to name this and the other session log commands with using "session", as in getSessionLog, getSessionLogTypes, isSessionLogAvailable